host/tests/rmfAudio_L3_TestCases/rmfAudio_test01_primaryDataCapture.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+import time
+

```

Comment:
please write a test document for L3 like other modules 
https://github.com/rdkcentral/rdk-halif-test-device_settings/tree/develop/docs/pages/dsAudio

Reply:
Added L3 test procedure document, and L3 low level spec..